director/commands/celery.py

```diff
@@ -38,9 +38,9 @@ def worker(dev_mode, worker_args):
     """Start a Celery worker instance"""
     args = [
         "celery",
-        "worker",
         "-A",
         "director._auto:cel",
+        "worker",
```

Comment:
Same question about the attributes order.

Reply:
Hello,

Same think my previous comment.